samples/openapi3/client/petstore/python-experimental/tests/test_deserialization.py

```diff
@@ -161,3 +161,26 @@ def test_deserialize_mammal(self):
         self.assertTrue(isinstance(deserialized, petstore_api.Zebra))
         self.assertEqual(deserialized.type, zebra_type)
         self.assertEqual(deserialized.class_name, class_name)
+
+    def test_deserialize_oneof_reference(self):
```

Comment:
Thanks for adding this test
SO the pattern that we have been using is to have this test_deserialization.py file verify deserializing json payloads into python classes.
We use the files like test_drawing.py to demonstrate that we can instantiate model instances correctly.

This test is helpful because it demonstrates that there is a bug here and drawings are not able to accept Shape/Triangle instances because type checking is failing.

In order to follow our patterns on where tests are how about
- moving this test into samples/openapi3/client/petstore/python-experimental/test/test_drawing.py ?

If you want to add a test here can you update it to demonstrate deserializing a Drawing json payload?

Reply:
done